addons/website/views/snippets/snippets.xml

```diff
@@ -1021,6 +1021,8 @@
             <we-button title="Show/hide logo" class="flex-grow-1 d-flex justify-content-center"
                     data-img="/website/static/src/img/snippets_options/header_extra_element_logo.svg"
                     data-customize-website-views="website.option_header_brand_logo"
+                    data-extra-possible-values="website.option_header_brand_name"
```

Comment:
Note: couldn't you add this in `customize-website-views` in customize-views data attr?

Reply:
If I understand what @rdeodoo means, I never tried it but I think this may work:

```xml
data-customize-website-views="|website.option_header_brand_name|website.option_header_brand_logo"
```
(see the two "|")

That would basically say there are 3 possible values: "", "name" and "logo". As it is a button, it has only two states though: the active one and the non active one. The active one uses the last possible value, hence "logo". The non active one use the first possible value, hence "".

Note that `data-customize-views="X"` is basically a shortcut for  `data-customize-views="|X"` in case of a button. Sometimes we have cases of `data-customize-views="X|"` for checkboxes: "enable this view when this checkbox is disabled".